app/src/main/java/com/example/myjokes/ui/joke_details/JokeDetailsPresenter.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.myjokes.ui.joke_details
+
+import com.example.myjokes.data.JokeGenerator
+
+class JokeDetailsPresenter(private val view: JokeDetailsView) {
+
+    fun loadJokeDetails(jokePosition: Int) {
+        if (jokePosition == -1) {
```

Comment:
Норм, но можно обработать и большее. 
```suggestion
        val joke = JokeGenerator.data.getOrNull(jokePosition)
        if (joke == null) { ... } else { ... }
```

Reply:
Ну или через try / catch